File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        auto lastUniqueIt = nums.begin();
        int n = nums.size();
        for (int i = 0; i<n; i++){
            if (nums[i] != *lastUniqueIt) *(++lastUniqueIt) = nums[i];
        }
        nums.erase(lastUniqueIt+1, nums.end());
        return nums.size();
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        l = len(nums)
        checker = nums[0]
        i = 1
        while i < l:
            if checker == nums[i]:
                nums.pop(i)
                l-=1
            else:
                checker = nums[i]
                i+=1
        return l
        